Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -631,5 +631,20 @@ private void AllView_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e)
             tapDebounceTimer.Stop();
             ParentShellPageInstance.InteractionOperations.OpenItem_Click(null, null);
         }
+
+        public async void PinItemToStart_Click(object sender, RoutedEventArgs e)
```

Comment:
Common between gridview and listview, can be moved to BaseLayout?

Reply:
@d2dyno1 asked me to move it here. Any objections to me moving it to BaseLayout?

Reply:
BaseLayout should be good (issue was putting it inside Interactions.cs) -> @d2dyno1 confirm?

Reply:
Will be ok